test/test-v8debugapi.js

```diff
@@ -43,6 +45,67 @@ function stateIsClean(api) {
   return true;
 }
 
+function validateVariable(variable) {
```

Comment:
Ok, we can consider this later. Especially since this validates data we get from V8 as opposed to user inputs.